site/kubernetes.md

```diff
@@ -0,0 +1,90 @@
+## Configuring Oracle WebLogic Server Kubernetes Operator
+
+The Extract Domain Resource Tool can be used to create a domain resource file for use with the Oracle WebLogic Server Kubernetes Operator. This allows the domain configuration and the Kubernetes container configuration to be specified in a single model file.
```

Comment:
We should emphasize that this specially interesting when making configuration changes that are also need to be reflected in the domain Custom Resource.  Example when adding a cluster to the domain just make the changes to the WDT model section that represents the WLS configuration and you can generate a Customer Resource yaml which you can apply to kubernetes.

Reply:
I'll add this, borrowing heavily from this comment!